### DISCUSSION TITLE
[Help]: Autoscroll Snapping

### DISCUSSION THREAD

**Discussion by mattbayX** [davidjerleke/embla-carousel#1004] [Help] [answered]
### Summary

Hi, I am having issues with auto-scroll for some reason. When there are a lot of logos, it starts snapping after a certain amount of time. If there are under 21 logos, it works fine, but this issue occurs when there are more. mobile version works fine just desktop version

Please see the video below and my code.

https://github.com/user-attachments/assets/5178f5ce-6b4e-4046-bc59-ac94a01a732e

`document.addEventListener('DOMContentLoaded', function () {

    const emblaNode = document.querySelector('.embla')

    if (emblaNode) {

        const OPTIONS = { axis: 'y', loop: true }
        const OPTIONS_MOBILE = { loop: true }

        const viewportNode = emblaNode.querySelector('.embla__viewport')
        const emblaDesktop = EmblaCarousel(viewportNode, OPTIONS, [AutoScroll()])

        const emblaNodeMobile = document.querySelector('.embla--mobile')
        const viewportNodeMobile = emblaNodeMobile.querySelector('.embla__viewport--mobile')

        const emblaMobile = EmblaCarousel(viewportNodeMobile, OPTIONS_MOBILE, [AutoScroll()])

    }

});`

### If applicable, which variants of Embla Carousel are relevant to this question?

- [X] embla-carousel **(Core)**
- [ ] embla-carousel-react
- [ ] embla-carousel-vue
- [ ] embla-carousel-svelte
- [ ] embla-carousel-autoplay
- [X] embla-carousel-auto-scroll
- [ ] embla-carousel-solid
- [ ] embla-carousel-auto-height
- [ ] embla-carousel-class-names
- [ ] embla-carousel-fade
- [ ] embla-carousel-docs **(Documentation)**
- [ ] embla-carousel-docs **(Generator)**

**Comment by davidjerleke** [comment]
Hi @mattbayX,

Please provide a CodeSandbox with your setup along with steps to reproduce the problem.

**Reply by mattbayX** [reply]
Yeah, Sure I have set one up here https://codesandbox.io/p/sandbox/brave-cori-sg6tw8 it seems to be when there's lot of logos i.e 21 or more it's breaking and starts snapping after a certain amount of time

**Reply by davidjerleke** [reply]
@mattbayX can you reproduce the problem with many items in [this example](https://www.embla-carousel.com/examples/predefined/#auto-scroll)?

**Reply by mattbayX** [reply, marked as answer]
Hi @davidjerleke i think i sorted this now as that example was working so  issue was with the CSS and the height of mulity logos thanks for you help

**Reply by davidjerleke** [reply]
@mattbayX glad to hear that. Cheers 👍🏻.
